fdbserver/VersionedBTree.actor.cpp

```diff
@@ -9072,7 +9075,8 @@ TEST_CASE(":/redwood/performance/extentQueue") {
 	// Choose a large remapCleanupWindow to avoid popping the queue
 	state Version remapCleanupWindow = params.getInt("remapCleanupWindow").orDefault(1e16);
 	state int numEntries = params.getInt("numEntries").orDefault(10e6);
-	state int diskReadSize = params.getInt("diskReadSize").orDefault(33554432);
+	//state int diskReadSize = params.getInt("diskReadSize").orDefault(33554432);
+	state int concExtentReads = params.getInt("concurrentExtentReads").orDefault(SERVER_KNOBS->REDWOOD_EXTENT_CONCURRENT_READS);
```

Comment:
I'd rather we used a full descriptive variable name `concurrentExtentReads`.

Reply:
Done.